src/cli-sdk/package.json

```diff
@@ -39,19 +39,20 @@
     "@vltpkg/spec": "workspace:*",
     "@vltpkg/types": "workspace:*",
     "@vltpkg/url-open": "workspace:*",
+    "@vltpkg/vlt-json": "workspace:*",
     "@vltpkg/vlx": "workspace:*",
     "@vltpkg/workspaces": "workspace:*",
     "@vltpkg/xdg": "workspace:*",
     "ansi-to-pre": "^1.0.6",
-    "ink": "^5.2.0",
     "ink-spinner": "^5.0.0",
+    "ink": "^5.2.0",
     "jackspeak": "^4.1.0",
     "package-json-from-dist": "catalog:",
     "path-scurry": "catalog:",
     "polite-json": "catalog:",
     "pretty-bytes": "^6.1.1",
-    "react": "^18.3.1",
     "react-devtools-core": "^4.28.5",
+    "react": "^18.3.1",
     "supports-color": "^10.0.0",
     "walk-up-path": "catalog:"
```

Comment:
`walk-up-path` is no longer being used in `cli-sdk` so i think this can be removed